core/secure_dt.mk

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Copyright (c) 2018, Linaro Limited
+# Copyright (c) 2015-2018, ARM Limited and Contributors. All rights reserved.
```

Comment:
ok, I understand. I'll notify you once I have some cleaner proposal.
For sake of simplicity, i will squash the current commits and append fixup changes on top for the rework you asked for.

Reply:
Great, +1 for squashing

Reply:
... things looks better when at the right place.
@jforissier i've created https://github.com/OP-TEE/optee_os/pull/2663 since your request fully modified this P-R current proposal.